hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -207,6 +211,24 @@ public byte[] getValueObject(
     }
   }
 
+  public byte[] getEndKeyValueObject(
```

Comment:
We can reuse getValueObject() passing startKey and endKey as parameter to this method.